tests/contrib/patch.py

```diff
@@ -65,18 +65,34 @@ def raise_if_no_attrs(f):
     required_attrs = [
```

Comment:
The changes made to this helper will be pulled into a separate PR once things are stable. This is the first integration that will be using the patch test cases (yay!!! 😄) and a few changes were necessary to get things working nicely.

Reply:
👍 for a separate PR